examples/rpc/starknet/starknet_addInvokeTransaction.hurl

```diff
@@ -8,7 +8,7 @@ Content-Type: application/json
             "type": "INVOKE",
             "max_fee":"0xDEADB",
             "version":"0x1",
-            "nonce":"0x1",
+            "nonce":"0x0",
```

Comment:
I need nonce set to 0 to be able to send the transaction as the first transaction.
Not sure if it's a problem from my setup, or something I am missing.